### PR TITLE
Mention Prometheus API requirements

### PR DESCRIPTION
**Pull request by prymitive**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -7,7 +7,29 @@ has_children: true
...
+Prometheus server.
+You can run pint in "offline" if you:
+- Don't pass any configuration file to pint.
+- You pass configuration file to pint that **doesn't** contain any `prometheus` definition.
+- You pass `--offline` flag to `pint` command.
+
+Most checks included in pint will require sending queries to a running Prometheus server where
+those rules are, or would be, deployed.
+Those checks are enabled if you pass a configuration file to pint that includes at least one
+`prometheus` block will.
```
Comment (filippog) [inline docs/index.md:26]: stray "will" here?

File: docs/index.md [hunk 2]
```diff
@@ -7,7 +7,29 @@ has_children: true
...
+Checks might use various Prometheus
+[HTTP API endpoints](https://prometheus.io/docs/prometheus/latest/querying/api/) to retrive
```
Comment (filippog) [inline docs/index.md:28]: retrieve

File: docs/index.md [hunk 3]
```diff
@@ -7,7 +7,29 @@ has_children: true
...
+extra information, for example Prometheus configuration or metrics metadata.
+If you run pint against a different service, like [Thanos](https://thanos.io/) some checks
+might return an erorr, since not all Prometheus HTTP APIs are supported by it.
```
Comment (filippog) [inline docs/index.md:31]: error

File: docs/index.md [hunk 4]
```diff
@@ -7,7 +7,29 @@ has_children: true
...
+In that case you might want to disable failing checks in pint configuration file.
```
Comment (filippog) [inline docs/index.md:32]: Maybe add something like "failing checks will be reported as `pint_problem`" to point users in the right direction

Comment (prymitive) [reply docs/index.md:32]: fix in #576

**Pull request merged**